Build error if AssemblyName is changed

When I change the assembly name of my actor project, the build fails with the following error:
```
C:\src> dotnet msbuild

Microsoft (R) Build Engine version 15.7.179.6572 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  MyActor.Interfaces -> C:\src\MyActor.Interfaces\bin\Debug\netcoreapp2.0\win7-x64\MyProject.MyActor.Interfaces.dll
  MyActor -> C:\src\MyActor\bin\Debug\netcoreapp2.0\win7-x64\MyProject.MyActor.dll
C:\Users\user\.nuget\packages\microsoft.servicefabric.actors\3.1.283\build\netcoreapp2.0\Microsoft.ServiceFabric.Actors.targets(16,5): error MSB3030: Could not copy the file "C:\src\MyActor\bin\Debug\netcoreapp2.0\win7-x64\\MyActor.runtimeconfig.dev.json" because it was not found. [C:\src\MyActor\MyActor.csproj]
```

This is because [`Microsoft.ServiceFabric.Actors.targets` uses the `MSBuildProjectName` property](https://github.com/Microsoft/service-fabric-services-and-actors-dotnet/blob/b19956611ca2022c64551ccca54c50277e18422a/src/netstandard/FabActUtil/Microsoft.ServiceFabric.Actors.targets#L16) to calculate the name of that file, instead of using the `AssemblyName` one:

```xml
    <Copy SourceFiles="$(TargetDir)\$(MSBuildProjectName).runtimeconfig.dev.json" DestinationFiles="$(FabActUtilWorkingDir)\FabActUtil.runtimeconfig.dev.json" SkipUnchangedFiles="true" />
    <Copy SourceFiles="$(TargetDir)\$(MSBuildProjectName).deps.json" DestinationFiles="$(FabActUtilWorkingDir)\FabActUtil.deps.json" SkipUnchangedFiles="true" />
```

I believe that simply changing to use `$(AssemblyName)` instead would fix the problem. I have tested locally and it seems to work, but there may be hidden side-effects that I am not aware of. I will submit a pull request with this change.

--- a/src/netstandard/FabActUtil/Microsoft.ServiceFabric.Actors.targets
+++ b/src/netstandard/FabActUtil/Microsoft.ServiceFabric.Actors.targets
@@ -13,8 +13,8 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       <ActorServiceAssemblies Include="$(TargetDir)\*.exe" />
       <FabActUtilAssemblies Include="$(MSBuildThisFileDirectory)\*" />
     </ItemGroup>
-    <Copy SourceFiles="$(TargetDir)\$(MSBuildProjectName).runtimeconfig.dev.json" DestinationFiles="$(FabActUtilWorkingDir)\FabActUtil.runtimeconfig.dev.json" SkipUnchangedFiles="true" />
-    <Copy SourceFiles="$(TargetDir)\$(MSBuildProjectName).deps.json" DestinationFiles="$(FabActUtilWorkingDir)\FabActUtil.deps.json" SkipUnchangedFiles="true" />
+    <Copy SourceFiles="$(TargetDir)\$(AssemblyName).runtimeconfig.dev.json" DestinationFiles="$(FabActUtilWorkingDir)\FabActUtil.runtimeconfig.dev.json" SkipUnchangedFiles="true" />
+    <Copy SourceFiles="$(TargetDir)\$(AssemblyName).deps.json" DestinationFiles="$(FabActUtilWorkingDir)\FabActUtil.deps.json" SkipUnchangedFiles="true" />
     <Copy SourceFiles="@(ActorServiceAssemblies)" DestinationFolder="$(FabActUtilWorkingDir)" SkipUnchangedFiles="true" />
     <Copy SourceFiles="@(FabActUtilAssemblies)" DestinationFolder="$(FabActUtilWorkingDir)" SkipUnchangedFiles="true" />
   </Target>

